src/Components/Suggestions/index.jsx

```diff
@@ -1,11 +1,13 @@
 import React from "react";
 import "./Suggestions.scss";
 
-const Suggestions = () => {
+const Suggestions = ({ searchResult }) => {
   return (
     <>
       <ul className="suggestion-list">
-        <li className="list-item">hi</li>
+        {searchResult.map((result) => (
+          <li key={result.word}>{result.word}</li>
+        ))}
```

Comment:
Same.

Reply:
Prettier automatically adds this parenthesis on save.